openerp/addons/base/res/res_partner_view.xml

```diff
@@ -182,7 +182,8 @@
                             <field name="fax"/>
                             <field name="user_ids" invisible="1"/>
                             <field name="email" widget="email" attrs="{'required': [('user_ids','!=', [])]}"/>
-                            <field name="title" options='{"no_open": True}'/>
+                            <field name="title" options='{"no_open": True}' string="Title" attrs="{'invisible': [('is_company','=',True)], 'readonly': [('is_company','=',True)]}" domain="[('domain','=','contact')]" context="{'default_domain': 'contact'}"/> 
+                            <field name="title" options='{"no_open": True}' string="Entity" attrs="{'invisible': [('is_company','=',False)], 'readonly': [('is_company','=',False)]}" domain="[('domain','=','partner')]" context="{'default_domain': 'partner'}"/>
```

Comment:
That will not work. I know it because I did it once and had to revert it ;-) bceb278199543dd9ed3fc00b76ef04a30786f19e